plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/EditConnectionWizard.java

```diff
@@ -100,6 +108,22 @@ public EditConnectionWizard(@NotNull DataSourceDescriptor dataSource) {
         });
     }
 
+    @NotNull
+    private static Map<String, String> getLibsIdVersion(@NotNull DataSourceDescriptor dataSource) {
+        Map<String, String> libs = new HashMap<>();
+        for (DBPDriverLibrary lib : getLibs(dataSource.getDriver())) {
+            libs.put(lib.getId(), lib.getVersion());
+        }
+        return libs;
+    }
+
+    @NotNull
+    private static List<? extends DBPDriverLibrary> getLibs(@NotNull DBPDriver driver) {
+        return driver instanceof DriverDescriptor driverDescriptor
+            ? driverDescriptor.getEnabledDriverLibraries()
+            : driver.getDriverLibraries();
```

Comment:
I just checked what it does, and it just filters the result of `getDriverLibraries()` by `DBPDriverLibrary#isDisabled`. You can remove the `instanceof` check and replicate this logic manually.

Reply:
Then you should probably rename `getLibs` itself to reflect what it does.